virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -109,6 +109,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-36165: VM disks with persistent reservation
 
 //CNV-36162: Search option in VM configuration tab
+* A virtual machine's configuration section now has a search field.
```

Comment:
The feature is about adding a search field in the VM configuration tab, so user can find the configure filed easily by searching, could you rephase it a bit? :)